### PR TITLE
fix: search in new multisig

### LINKED ISSUE #2485: Multisig. Search in create multisig apply to address book
## Description

When user search in multisig creation flow, it applies to Address Book

https://github.com/user-attachments/assets/f1d92886-5aaa-46f0-9375-bb50ae2931b0

## Environment
https://github.com/novasamatech/nova-spektr/commit/6ca515af94bd4d44ea25ada390bcc01455cd4bc4

### PR DESCRIPTION
**Pull request by Tbaut**
closes https://github.com/novasamatech/nova-spektr/issues/2485

image

### CODE REVIEW

File: src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/Signatory.tsx [hunk 1]
```diff
@@ -34,13 +34,44 @@ export const Signatory = ({
...
   const wallets = useUnit(walletModel.$wallets);
   const {
     fields: { chain },
   } = useForm(formModel.$createMultisigForm);
+  const contactsFiltered = useMemo(
+    () =>
+      performSearch({
+        query,
+        records: contacts,
+        getMeta: ({ id, name, address, accountId }) => ({
```
Comment (johnthecat) [inline src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/Signatory.tsx]: This method should map additional values, you can safely remove it

File: src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/Signatory.tsx [hunk 2]
```diff
@@ -34,13 +34,44 @@ export const Signatory = ({
   const [query, setQuery] = useState('');
   const [options, setOptions] = useState<ComboboxOption[]>([]);
 
-  const [contacts, contactsFiltered] = useUnit([contactModel.$contacts, filterModel.$contactsFiltered]);
+  const [contacts] = useUnit([contactModel.$contacts]);
```
Comment (johnthecat) [inline src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/Signatory.tsx]:
```suggestion
  const contacts = useUnit(contactModel.$contacts);
```

File: src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/Signatory.tsx [hunk 3]
```diff
@@ -34,13 +34,44 @@ export const Signatory = ({
...
+        }),
+        weights: {
+          name: 1,
+          address: 0.5,
+        },
+      }),
+    [query, contacts],
+  );
+
+  // performSearch({
```
Comment (johnthecat) [inline src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/Signatory.tsx]: Why?

Comment (Tbaut) [reply src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/Signatory.tsx]: sorry, forgot to push to remove it. Just did along with fixing what you suggested :pray:

**Review by johnthecat — APPROVED** [review]

**Review by Asmadek — APPROVED** [review]

**Review by sokolova-an — APPROVED** [review]

**Pull request merged**